### PR TITLE
Create 2023-09-15-Remitano.md

### PR DESCRIPTION
**Pull request by svg-arch**

### CODE REVIEW

File: content/attacks/posts/2023-09-15-Remitano.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Attackers
+
+The identity of the attacker is unknown. The following addresses are associated with this attack:
+
+   - [0x74530e81e9f4715c720b6b237f682cd0e298b66c](https://etherscan.io/address/0x74530e81e9f4715c720b6b237f682cd0e298b66c)
+   - [TEDNf1aqk8YJEUdNH9NRd4MqibZmdP49Fm](https://tronscan.org/#/address/TEDNf1aqk8YJEUdNH9NRd4MqibZmdP49Fm)
+
+## Losses
+
+Remitano lost approximately $2,700,000 during the attack and $1.9 million was frozen by Tezer. The drainer swapped the stolen USDC and ANKR for 163 ETH (264000 USD) and [transferred them to HitBTC](https://twitter.com/PeckShieldAlert/status/1702507801669775712).
```
Comment (JediFaust) [inline content/attacks/posts/2023-09-15-Remitano.md:26]: There is typo: `Tezer`

File: content/attacks/posts/2023-09-15-Remitano.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Assets stolen on ETH:
```
Comment (JediFaust) [inline content/attacks/posts/2023-09-15-Remitano.md:28]: ETH is a cryptocurrency, not a chain. So write as Ethereum instead.

File: content/attacks/posts/2023-09-15-Remitano.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+- [1,359,253 USDT](https://etherscan.io/tx/0xe0725362fd774de0d8416d5e3d028063508ffa61f68087c576320e42159677a9)
+- [208,188 USDC](https://etherscan.io/tx/0x574a416cf868e3439da278ae7043c7eb364a76d323dddf7c5b1c155628843c5f)
+- [34.4 ETH](https://etherscan.io/tx/0xef9a766c6a7a935ee26f4d1e36cd7c19a14b84559c2d350be3aca217273c7891)
+- [104,360 ANKR](https://etherscan.io/tx/0x5fb5119be8b588edf198fd2de73ba8e5a3758474176be7e721f5cb841009f532)
+
+Assets stolen on Tron:
+
+- [537,915 USDT](https://tronscan.org/#/address/TEDNf1aqk8YJEUdNH9NRd4MqibZmdP49Fm/transfers)
```
Comment (JediFaust) [inline content/attacks/posts/2023-09-15-Remitano.md:37]: The link refers to an address, not a transaction. Replace with a transaction link as in the link below.

File: content/attacks/posts/2023-09-15-Remitano.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- [3,750,700 TRX](https://tronscan.org/#/transaction/4792706c53504495863c7bf23b15922e36afd8b97a2e66c0c2b1dfab1d2e0e9f)
+
+## Timeline
+
+- **2023-09-14, 03:48:59 PM UTC:** [Initial malicious transaction](https://etherscan.io/tx/0x99ad9f25df9c2597de9c8ac159f7f4d4f7139ffba0da945c4ee088f33da274e8) occured.
```
Comment (JediFaust) [inline content/attacks/posts/2023-09-15-Remitano.md:42]: Fix timeline formatting as shown in other articles.

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2023-09-15-Remitano.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+- **September 14, 2023, 03:48:59 PM UTC:** [Initial malicious transaction](https://etherscan.io/tx/0x99ad9f25df9c2597de9c8ac159f7f4d4f7139ffba0da945c4ee088f33da274e8) occured.
+- **September 14, 2023, 10:46 PM UTC:** Suspicious withdrawals were detected by [Cyvers](https://twitter.com/CyversAlerts/status/1702348063145165016)   
+- **September 15, 2023, 04:35 PM UTC:** Remitano [announces](https://twitter.com/remitano/status/1702617066761826794) security vulnerability in an X post.
+- **September 15, 2023:** Remitano [releases updated recovery plan](https://remitano.com/forum/ng/134684-latest-updates-on-recent-security-incident-on-remitano).
+- **September 15, 2023:** Immunebytes [published](https://www.immunebytes.com/blog/remitano-crypto-exchange-lost-2-7-millions-in-an-alleged-exploit) a detailed analysis of the incident.
+
+## Security Failure Causes
+
+**Compromised Private Key:** The breach was initiated through a data compromise from a third-party source, exposing sensitive information about the exchange. This security lapse enabled unauthorized transactions, resulting in the illicit transfer of funds from the exchange's hot wallets to suspicious addresses.
```
Comment (JediFaust) [inline content/attacks/posts/2023-09-15-Remitano.md:50]: Fix formatting please, make it a point

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by evgenydmitriev — APPROVED** [review]

**Pull request merged**